modules/jdbc/src/integrationTest/java/org/apache/ignite/jdbc/ItJdbcMultiStatementSelfTest.java

```diff
@@ -417,7 +430,7 @@ public void testAutoCommitFalse() throws Exception {
     @Test
     @SuppressWarnings("ThrowableNotThrown")
     public void testAutoCommitFalseTxControlStatementsNotSupported() throws Exception {
-        String txErrMsg = "Transaction control statements are not supported when autocommit mode is disabled";
+        String txErrMsg = "Transaction control statement cannot be executed within an external transaction.";
```

Comment:
The text of this exception was previously specifically remapped 
https://issues.apache.org/jira/browse/IGNITE-21167

If it's not clear how to do this - file the ticket to fix this message.

p.s. maybe we can just change the original exception message, need to discuss